src/main/scala/firrtl/Driver.scala

```diff
@@ -90,7 +92,20 @@ object Driver {
       val annotationFile = new File(annotationFileName)
       if (annotationFile.exists) {
         val annotationsYaml = io.Source.fromFile(annotationFile).getLines().mkString("\n").parseYaml
-        val annotationArray = annotationsYaml.convertTo[Array[Annotation]]
+        val annotationArray = {
+          val annos = annotationsYaml.convertTo[Array[Annotation]]
+
+          if (annos.nonEmpty) {
+            annos ++ List(Annotation(
+              ModuleName("All", CircuitName("All")),
```

Comment:
@azidar Will fix.  Problem is that this annotation does not even target a module at all.  Changed for now to CircuitName("All")
probably better but would take more time to follow through would be to create
```scala
final case class NoCircuit extends Named {
    def serialize: String = "NoCircuitSpecified"
}
```